src/software/ai/hl/stp/tactic/crease_defender/crease_defender_fsm.cpp

```diff
@@ -91,18 +106,39 @@ void CreaseDefenderFSM::blockThreat(
     TbotsProto::BallCollisionType ball_collision_type =
         TbotsProto::BallCollisionType::ALLOW;
     if ((event.common.world_ptr->ball().position() - destination).length() <
-        (event.common.robot.position() - destination).length())
+        (robot_position - destination).length())
     {
         ball_collision_type = TbotsProto::BallCollisionType::AVOID;
     }
 
+    AutoChipOrKick auto_chip_or_kick{AutoChipOrKickMode::OFF, 0};
+    auto goal_line_segment =
+        Segment(event.common.world_ptr->field().friendlyGoal().posXPosYCorner(),
+                event.common.world_ptr->field().friendlyGoal().posXNegYCorner());
```

Comment:
I still feel like we should at least add some buffer to this value (e.g. extending it up and down by 0.5 meters), or using the segment that makes the left sideline of the field. Having the threshold be the exact size of the goal is really putting a lot of trust in our ability to be able to always chip exactly so we don't end up chipping into our own net.

Reply:
Sure, I will add a buffer to the upper and lower net bounds